### PR TITLE
Merge beta.7 release branch to master

### PR DESCRIPTION
**Pull request by daxian-dbw**
Merge beta.7 release branch to master. Make sure choose the option "**Create a merge commit**" to merge this PR, so the tag can be easily synced to clients with a fetch.

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Review by mirichmo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ You can download and install a PowerShell package for any of the following platf
...
-[rl-windows7-64]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/PowerShell-6.0.0-beta.6-win7-win2008r2-x64.msi
-[rl-windows7-86]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/PowerShell-6.0.0-beta.6-win7-x86.msi
-[rl-ubuntu16]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/powershell_6.0.0-beta.6-1ubuntu1.16.04.1_amd64.deb
-[rl-ubuntu14]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/powershell_6.0.0-beta.6-1ubuntu1.14.04.1_amd64.deb
-[rl-centos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/powershell-6.0.0_beta.6-1.el7.x86_64.rpm
-[rl-ai]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/PowerShell-6.0.0-beta.6-x86_64.AppImage
-[rl-macos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/powershell-6.0.0-beta.6-osx.10.12-x64.pkg
+[rl-windows10]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win10-win2016-x64.msi
```
Comment (adityapatwardhan) [inline README.md]: This needs to be updated for the single package name.

File: README.md [hunk 2]
```diff
@@ -43,15 +43,15 @@ You can download and install a PowerShell package for any of the following platf
...
+[rl-windows81]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win81-win2012r2-x64.msi
```
Comment (adityapatwardhan) [inline README.md]: Similarly, this needs to be updated for the single package name.

File: README.md [hunk 3]
```diff
@@ -43,15 +43,15 @@ You can download and install a PowerShell package for any of the following platf
...
+[rl-windows7-64]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win7-win2008r2-x64.msi
```
Comment (adityapatwardhan) [inline README.md]: Same here.

File: README.md [hunk 4]
```diff
@@ -43,15 +43,15 @@ You can download and install a PowerShell package for any of the following platf
...
+[rl-windows7-86]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win7-x86.msi
```
Comment (adityapatwardhan) [inline README.md]: Same here.

File: docker/release/nanoserver/Dockerfile [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # escape=`
 FROM microsoft/nanoserver:latest
 
-ARG POWERSHELL_ZIP=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/PowerShell-6.0.0-beta.6-win10-win2016-x64.zip
-ARG POWERSHELL_VERSION=6.0.0-beta.6
+ARG POWERSHELL_ZIP=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win10-win2016-x64.zip
```
Comment (adityapatwardhan) [inline docker/release/nanoserver/Dockerfile]: Similarly, this needs to be updated for the single package name.

File: docker/release/nanoserver-insider/Dockerfile [hunk 6]
```diff
@@ -10,7 +10,7 @@ ARG NanoServerRepo=microsoft/nanoserver-insider
 FROM ${WindowsServerCoreRepo}:$WindowsServerCoreVersion  AS installer-env
 
 # Arguments for installing powershell, must be defined in the container they are used
-ARG PS_VERSION=6.0.0-beta.6
+ARG PS_VERSION=6.0.0-beta.7
 
 ENV PS_DOWNLOAD_URL https://github.com/PowerShell/PowerShell/releases/download/v$PS_VERSION/PowerShell-$PS_VERSION-win10-win2016-x64.zip
```
Comment (adityapatwardhan) [inline docker/release/nanoserver-insider/Dockerfile]: Similarly, this needs to be updated for the single package name.

File: docker/release/windowsservercore/Dockerfile [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # escape=`
 FROM microsoft/windowsservercore:latest
 
-ARG POWERSHELL_MSI=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.6/PowerShell-6.0.0-beta.6-win10-win2016-x64.msi
-ARG POWERSHELL_VERSION=6.0.0-beta.6
+ARG POWERSHELL_MSI=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.7/PowerShell-6.0.0-beta.7-win10-win2016-x64.msi
```
Comment (adityapatwardhan) [inline docker/release/windowsservercore/Dockerfile]: Similarly, this needs to be updated for the single package name.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
The windows package names are updated. Please take another look. It's easier to just review the last commit.

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Review by TravisEz13 — APPROVED** [review]

**Review by mirichmo — APPROVED** [review]

**Pull request merged**